server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -41,6 +42,20 @@ func NewBuildBuddyServer(env environment.Env, sslService *ssl.SSLService) (*Buil
 	}, nil
 }
 
+func (s *BuildBuddyServer) getSanitizedGetInvocationResponse(ctx context.Context, unsanitizedResponse *inpb.GetInvocationResponse) (*inpb.GetInvocationResponse, error) {
```

Comment:
how bout calling this "redactAPIKey"

Reply:
SG, "sanitize" is probably a bit abstract for what this is doing.